caltechdata_api/cli.py

```diff
@@ -401,25 +415,34 @@ def create_record():
             if existing_data:
                 if filepath != "":
                     response = caltechdata_write(
-                        existing_data, token, filepath, production=True, publish=False
+                        existing_data, token, filepath, production=production, publish=False
                     )
                 elif file_link != "":
                     response = caltechdata_write(
                         existing_data,
                         token,
                         file_links=[file_link],
                         s3_link=file_link,
-                        production=True,
+                        production=production,
                         publish=False,
                     )
                 else:
                     response = caltechdata_write(
-                        existing_data, token, production=True, publish=False
+                        existing_data, token, production=production, publish=False
                     )
                 rec_id = response
-                print(
+                if production == True: 
+

```

Comment:
I would set a variable for the URL based on production. That way we can have just one copy of the message